src/main/java/HandlerAssembler.java

```diff
@@ -6,6 +6,8 @@ public List getAllHandlers() {
         List<Handler> handlers = new ArrayList<>();
```

Comment:
Very minor but could be worth looking at something like Arrays.asList to simplify the initialisation.

At the moment this list is rebuilt every time the function is called - depending on how it is used could the function be replaced with a constant?